otsdaq/CoreSupervisors/CorePropertySupervisorBase.cc

```diff
@@ -509,8 +509,7 @@ void CorePropertySupervisorBase::getRequestUserInfo(WebUsers::RequestUserInfo& u
 {
 	checkSupervisorPropertySetup();
 
-	//__SUP_COUT__ << "userInfo.requestType_ " << userInfo.requestType_ << " files: " <<
-	// cgiIn.getFiles().size() << __E__;
+	__SUP_COUT__ << "userInfo.requestType_ " << userInfo.requestType_ << __E__;
```

Comment:
Please comment these printouts (line 512, 544-548) and also in WebUsers, put back to COUTT